### ISSUE TITLE
RI events

### ISSUE THREAD

**Issue by limeng12** [YeoLab/outrigger#90]
Dear author,

I think it's a fantastic idea to use the graph to call alternative splicing events. Do you have plans to continue to maintain and upgrade the software? I ask because I see it hasn't been updated for a long time. I want to use this, but it missed several important AS events, like A5SS, A3SS, RI events, currently only SE and MXE are included. So I think if you can provide some ideas for designing this software. So if you don't have plans to upgrade it, I can add these functions. 

Meng

**Comment by geneyeo** [comment]
Yes we are hoping to maintain and upgrade the software - but if you are interested to contribute to annotate specific events, we can discuss that would be great.
Gene Yeo, PhD, MBA
Professor
Dept of Cellular and Molecular Medicine
University of California, San Diego
Institute for Genomic Medicine
UCSD Stem Cell Program
geneyeo@ucsd.edu


Yeo Lab Mailing address:
2880 Torrey Pines Scenic Drive
Sanford Consortium for Regenerative Medicine
Room 3008, Gene’s Office
Room 3805, Wet Lab
Room 3114/3115, Bioinformatics Lab
La Jolla, CA 92037
   
Lab website: http://yeolab.github.io

**Comment by limeng12** [comment]
Dear authors,

I noticed that this software is built using python, which is not in my programming skillset (R, Java). So I used other software instead. My research interest is analyzing data and splicing, I think maybe I can contribute some codes/ideas if you have some Github/research projects that analyzing datasets in the future. I also hope that the outrigger project can succeed.

Best wishes,
Meng
